Sources/RingPublishingTracking/Private/Tracking/Events/EventsService.swift

```diff
@@ -101,26 +125,25 @@ final class EventsService {
     }
 
     /// Adds list of events to the queue when the size of each event is appropriate
+    ///
     /// - Parameter events: Array of `Event` that should be added to the queue
     func addEvents(_ events: [Event]) {
-        let decoratedEvents: [Event] = events.map { event in
-            event.decorated(using: decorators)
-        }
+        decorateEvents(events, using: decorators) { [weak self] decoratedEvents in
```

Comment:
jedyne co mnie zastanawia to ten async teraz. 
- jaki był jego powód
- czy niczego istotnego nie zmieni w kontekście momentu wysyłki eventu?

Reply:
bo nigdzie nie mówimy czy moduł może być używany z bg thread'a czy nie - a jak był to odwołanie do window sypie warningiem, że to nie jest z wątku głównego

tylko jest dispatch by dekoratory zawołać a potem jest jak było, kolejka zdarzeń, nic w wysyłce sie nie zmienia